index.js

```diff
@@ -1,24 +1,50 @@
 var chalk = require('chalk');
 
+var buildPolicyString = require('./lib/utils')['buildPolicyString'];
+
+var CSP_SELF        = "'self'";
+var CSP_NONE        = "'none'";
+var CSP_UNSAFE_EVAL = "'unsafe-eval'";
+var REPORT_PATH     = '/csp-report';
+
+var CSP_HEADER              = 'Content-Security-Policy';
+var CSP_HEADER_REPORT_ONLY  = 'Content-Security-Policy-Report-Only';
+
+var CSP_REPORT_URI          = 'report-uri';
+var CSP_FRAME_ANCESTORS     = 'frame-ancestors';
+var CSP_SANDBOX             = 'sandbox';
+
+var META_UNSUPPORTED_DIRECTIVES = [
+  CSP_REPORT_URI,
+  CSP_FRAME_ANCESTORS,
+  CSP_SANDBOX,
+];
+
+var unsupportedDirectives = function(policyObject) {
+  return META_UNSUPPORTED_DIRECTIVES.filter(function(name) {
+    return policyObject && (name in policyObject);
+  });
+}
+
 module.exports = {
   name: 'ember-cli-content-security-policy',
 
   config: function(environment /*, appConfig */) {
     var ENV = {
-      contentSecurityPolicyHeader: 'Content-Security-Policy-Report-Only',
+      contentSecurityPolicyHeader: CSP_HEADER_REPORT_ONLY,
       contentSecurityPolicy: {
-        'default-src': "'none'",
-        'script-src': "'self'",
-        'font-src': "'self'",
-        'connect-src': "'self'",
-        'img-src': "'self'",
-        'style-src': "'self'",
-        'media-src': "'self'"
+        'default-src':  CSP_NONE,
+        'script-src':   CSP_SELF,
+        'font-src':     CSP_SELF,
+        'connect-src':  CSP_SELF,
+        'img-src':      CSP_SELF,
+        'style-src':    CSP_SELF,
+        'media-src':    CSP_SELF
       }
     };
 
     if (environment === 'development') {
-      ENV.contentSecurityPolicy['script-src'] = ENV.contentSecurityPolicy['script-src'] + " 'unsafe-eval'";
+      ENV.contentSecurityPolicy['script-src'] += (' ' + CSP_UNSAFE_EVAL);
```

Comment:
I don't think this is needed anymore. Ember CLI hasn't used evaled strings (for fake source maps) for a very long time...